src/Sulu/Component/Webspace/Analyzer/Attributes/WebsiteRequestProcessor.php

```diff
@@ -70,6 +86,11 @@ function (PortalInformation $a, PortalInformation $b) {
 
         /** @var PortalInformation $portalInformation */
         $portalInformation = reset($portalInformations);
+        foreach ($portalInformations as $information) {
```

Comment:
Maybe this could also be part of the usort before, so that the simple `reset` line is even enough?